src/error_messages/error_messages.h

```diff
@@ -252,6 +252,7 @@
 #define AG_NOKEYS_EXIT  "(4109): Unable to start without auth keys. Exiting."
 #define AG_MAX_ERROR    "(4110): Maximum number of agents '%d' reached."
 #define AG_AX_AGENTS    "(4111): Maximum number of agents allowed: '%d'."
+#define AG_INV_MNGIP    "(4112): No valid server address found."
```

Comment:
Better to say: `Invalid manager addess found.` + the value of that invalid address. Yo can always print `agt->server[0]` as the invalid IP due to if it would not the invalid IP, `Validate_Address` won't fail